SpanishFilter/sections/antiadblock.txt

```diff
@@ -11,6 +11,8 @@ tunovelaligera.com#@#iframe[width="728"][height="90"]
 tunovelaligera.com#@#img[width="728"][height="90"]
 !#endif
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/73853
+||www.anime-jl.net/animeflv/js/detector.js
```

Comment:
I don't see the request. Do you still have it? If so, please provide a screenshot.

Reply:
Yep they removed it. Probably they saw this PR -_-.